ironfish/src/rpc/adapters/errors.ts

```diff
@@ -9,6 +9,7 @@ export enum ERROR_CODES {
   ROUTE_NOT_FOUND = 'route-not-found',
   VALIDATION = 'validation',
   INSUFFICIENT_BALANCE = 'insufficient-balance',
+  UNAUTHENTICATION = 'unauthentication',
```

Comment:
nit - `unauthentication` looks odd. Maybe `unauthenticated` or `unauthorized`?

Reply:
Yes we should copy HTTP here probably which is `Unauthorized`, https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401

Reply:
I will use `unauthenticated` 
HTTP code is technically not accurate, see the post here http://web.archive.org/web/20190904190534/https://www.dirv.me/blog/2011/07/18/understanding-403-forbidden/index.html
`Unauthorized` should be used when server know who are the user of the request, but the user don’t have permission to access this resource.